servers/src/main/java/tachyon/worker/tiered/StorageDir.java

```diff
@@ -494,10 +494,12 @@ public long getLastBlockAccessTimeMs(long blockId) {
    */
   public long getLockedSizeBytes() {
     long lockedBytes = 0;
-    for (long blockId : mUserPerLockedBlock.keySet()) {
-      Long blockSize = mBlockSizes.get(blockId);
-      if (blockSize != null) {
-        lockedBytes += blockSize;
+    synchronized (mUserPerLockedBlock) {
```

Comment:
Is this the same "lock" that the internal methods of 'mUserPerLockedBlock' use? I am not familiar with Multimaps.synchronizedMultimap().